mantis-discovery-proto/build.gradle

```diff
@@ -15,7 +15,7 @@
  */
 
 dependencies {
-    api project(path: ':mantis-shaded', configuration: 'shadow')
+    api "io.mantisrx:mantis-shaded:${rootProject.versionMantisShaded}"
```

Comment:
this should be implementation probably?

Reply:
I'm doing a like-for-like replacement at the moment, will do this in a separate PR.